runtime/Go/antlr/interval_set.go

```diff
@@ -23,7 +23,15 @@ func NewInterval(start, stop int) *Interval {
 	return i
 }
 
-func (i *Interval) contains(item int) bool {
+func (i *Interval) GetStart() int {
```

Comment:
I agree about Go style doesn't use `Get`, I just respect consistency among other codes. And other way is just expose fields, as you said.

So,

1. Change to `Start()` and `Stop()`
2. Expose directory (change `start int` to `Start int` in `Interval` struct

Which is better for this project? I will follow your comment. (personally, 2 is prefer for me)